src/Uno.UWP/Storage/Pickers/FileSavePicker.macOS.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using AppKit;
+using Foundation;
+using Windows.Foundation;
+
+namespace Windows.Storage.Pickers
+{
+	public partial class FileSavePicker
+	{
+		public PickerLocationId SuggestedStartLocation { get; set; }
```

Comment:
`SuggestedStartLocation` should be shared by all implementations. Please move to `FileSavePicker.cs`

Reply:
ah, ok. I assumed it would be implemented on all as it is an enum property. Shouldn't we just flag it as implemented for all platforms at once? Of course the value can be ignored on a per platform basis.

Reply:
We should probably have some clearer directives on such implementations, but for now, properties have more weight than enum values (as enums could be used in different classes). Marking a property implemented just for the platform makes sense here, as it is only used for two platforms (and it's only an autoproperty).